pkg/sql/conn_executor_exec.go

```diff
@@ -2241,11 +2257,13 @@ func (ex *connExecutor) handleTxnRowsWrittenReadLimits(ctx context.Context) erro
 // transaction contains multiple statements, and an upgrade was attempted, an
 // error is returned.
 func (ex *connExecutor) maybeUpgradeToSerializable(ctx context.Context, stmt Statement) error {
+	var txnSchemaChangeErr = pgerror.Newf(
```

Comment:
This should be defined globally as opposed to in this function (which will allocate a new one every time)